src/elements/MultiToggleSwitch.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react'
+import { makeStyles } from '@material-ui/styles'
+import clsx from 'clsx'
+import Theme from '../theme-manager/themes/model'
+import Text from './Text'
+
+interface MultiToggleSwitchProps {
+  options: Array<{ label: string; value: unknown }>
+  onChange?: (value) => void
+  className?: string
+  activeOptionColor?: string
+}
+
+export default function MultiToggleSwitch({
+  options = [],
+  onChange,
+  className,
+  activeOptionColor,
+}: MultiToggleSwitchProps) {
+  const classes = useStyles({ activeOptionColor })
+  const [activeOption, setActiveOption] = useState(options[0]?.value)
```

Comment:
may be a good idea to make this a controlled component (pass in value as a prop & have the switch update based on that) so that we could alter the toggle switch outside of the component